Source/Codecov/Upload/CodecovUploader.cs

```diff
@@ -42,13 +42,9 @@ protected virtual void ConfigureContent(HttpContent content)
             content.Headers.ContentType = new MediaTypeHeaderValue("application/x-gzip");
         }
 
-        protected virtual void ConfigureRequest(HttpRequestMessage request)
```

Comment:
Unfortunately, this method can not be removed entirely, as it is overridden in the fallback uploader to set the needed headers for the request there.

You could change the method to be an empty statement instead:
```cs
protected virtual void ConfigureRequest(HttpRequestMessage request) { }
```

Reply:
Modified this PR to take your recommendation, here. Please let me know if that's sufficient.